usaspending_api/api_contracts/contracts/search/AdvancedSearch.md

```diff
@@ -17,18 +17,18 @@ This endpoint takes award filters and fields, and returns the fields of the filt
 
 + Request (application/json)
     + Attributes (object)
-        + filters (optional, FilterObject)
-        + fields (optional, SpendingByAwardFields)
+        + `filters` (optional, FilterObject)
+        + `fields` (required, SpendingByAwardFields)
```

Comment:
Was this intentional?

Reply:
Yes
https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/search/v2/views/search.py#L43